django/core/paginator.py

```diff
@@ -17,13 +18,14 @@ class EmptyPage(InvalidPage):
 
 
 class Paginator(object):
-
     def __init__(self, object_list, per_page, orphans=0,
-                 allow_empty_first_page=True):
+                 allow_empty_first_page=True, request=None, page_kwarg=None):
         self.object_list = object_list
         self.per_page = int(per_page)
         self.orphans = int(orphans)
         self.allow_empty_first_page = allow_empty_first_page
+        self.request = request
+        self.page_kwarg = page_kwarg or 'page'
```

Comment:
is there a reason not to use `page_kwarg='page'` instead of the current technique?